db/change_cache.go

```diff
@@ -589,9 +589,10 @@ func (c *changeCache) processUnusedRange(ctx context.Context, fromSequence, toSe
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
+	var isEmpty bool
```

Comment:
i would rename this var removedSkippedSeqs to represent what this means better.

Reply:
This was a variable to indicate if the skipped list is empty or not after removal so I don't think that makes sense. 

This has now been changed to return actual count not boolean based on review above